include/sys/util.h

```diff
@@ -770,6 +770,92 @@ u8_t u8_to_dec(char *buf, u8_t buflen, u8_t value);
  */
 #define UTIL_LISTIFY(LEN, F, ...) UTIL_EVAL(UTIL_REPEAT(LEN, F, __VA_ARGS__))
 
+/**
+ * @brief Get number of items on a list. It works with lists up to
+ * 20 items.
+ *
+ * Example:
+ *    \#define LIST led0, led1, led5
+ *    int leds = UTIL_LIST_LEN(LIST);
+ *    will assign 3 to leds.
```

Comment:
```suggestion
 * Example:
 *
 *    #define LIST led0, led1, led5
 *    int leds = UTIL_LIST_LEN(LIST);
 *
 * will assign 3 to leds.
```

I spent some time searching, and I can't find anywhere this header is pulled into HTML via breathe. However, we might as well make it look good in Doxygen.